app/routes/user/permissions.tsx

```diff
@@ -0,0 +1,300 @@
+/*!
+ * Copyright © 2022 United States Government as represented by the Administrator
+ * of the National Aeronautics and Space Administration. No copyright is claimed
+ * in the United States under Title 17, U.S. Code. All Other Rights Reserved.
+ *
+ * SPDX-License-Identifier: NASA-1.3
+ */
+
+import type { DataFunctionArgs } from '@remix-run/node'
+import { Form, useFetcher, useLoaderData } from '@remix-run/react'
+import {
+  Button,
+  ButtonGroup,
+  ComboBox,
+  Grid,
+  Label,
+} from '@trussworks/react-uswds'
+import { useState } from 'react'
+import SegmentedCards from '~/components/SegmentedCards'
+import { getFormDataString } from '~/lib/utils'
+import { getUser } from '../__auth/user.server'
+import type { EndorsementRequest } from './endorsements.server'
+import { EndorsementsServer } from './endorsements.server'
+
+type intermediateMappedUser = {
+  sub: string
+  email: string
+  name: string
+}
+
+export async function loader({ request }: DataFunctionArgs) {
+  const endorsementServer = await EndorsementsServer.create(request)
+  const user = await getUser(request)
+
+  // Maps the retrived users to an object that can be used in the ComboBox
+  const validUsers = await endorsementServer
+    .getSubmitterUsers()
+    .then((response) => {
+      const mappedUsers = response?.map((user) =>
+        user.Attributes?.filter((attr) =>
+          ['sub', 'email', 'name'].includes(attr.Name as string)
+        ).reduce(
+          (obj, item) =>
+            Object.assign(obj, { [item.Name as string]: item.Value }),
+          {}
+        )
+      ) as intermediateMappedUser[]
+
+      return mappedUsers.map((user) => ({
+        value: user.sub,
+        label: `${user.name} - ${user.email}`,
```

Comment:
My preference would be to return the email, name, and sub from the loader, and let the React component do the formatting.

Reply:
I can make that work, I'll have that in the next update